index.d.ts

```diff
@@ -20,16 +20,16 @@ declare namespace hooks {
   }
 
   interface HookProps<T> {
+    app?: feathers.Application;
+    data?: T;
+    error?: any;
+    id?: string | number;
     method?: string;
-    type: 'before' | 'after' | 'error';
     params?: any;
-    data?: T;
+    path?: string;
     result?: T;
-    app?: feathers.Application;
-    service?: Service<T>;
-    
-    // server side only
-    id?: string;
```

Comment:
Since `Service` is outside this namespace, I think it should be `feathers.Service<T>`.

Reply:
good catch, thanks